File: programme/digital-root/digital_root.js
// Write a programme for Digital Root of a number

function digital_root(n) {
  if (n === 0) {
    return 0;
  } else {
    return 1 + ((n - 1) % 9);
  }
}

let num = prompt("Enter Number : ");
console.log("Digital Root :", digital_root(num));

File: programme/digital-root/digital_root.cpp
// Write a programme for Digital Root of a number
#include <iostream>

using namespace std;

int digital_root(int n)
{
    if (n == 0)
        return 0;
    else
    {
        return 1 + ((n - 1) % 9);
    }
}

int main()
{
    int n;
    cout << "Enter Number : ";
    cin >> n;
    cout << endl;
    cout << "Digital Root Value : " << digital_root(n) << endl;
    return 0;
}
